### ISSUE TITLE
if in if

### ISSUE THREAD

**Issue by larshp** [larshp/abaplint#394]
should catch

```abap
      ELSE.
        IF lv_bitbyte+lv_offset(1) = '1'.
          rv_int = rv_int + ( 2 ** ( sy-index - 1 ) ).
        ENDIF.
      ENDIF.
```
